### ISSUE TITLE
Voiceover

### ISSUE THREAD

**Issue by MichaelaRob** [AspirationYC/ecommerce-pattern-library#4]
when trying to enter the website with voiceover, unable to tab or access without clicking. Add in some aria details to make voiceover more accessible
